kernel/aster-nix/src/syscall/sched_getaffinity.rs

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use core::mem;
+
+use super::SyscallReturn;
+use crate::{
+    prelude::*,
+    process::{process_table, Pid},
+    util::write_val_to_user,
+};
+
+pub fn sys_sched_getaffinity(
+    pid: Pid,
+    cpuset_size: usize,
+    user_mask_ptr: Vaddr,
+) -> Result<SyscallReturn> {
+    // Ensure cpuset_size is sufficient to store a cpu_set_t.
```

Comment:
Please remove worthless comments. They do not provide any extra information or provide information in a more comprehensible way than the code.

```rust
// Ensure cpuset_size is sufficient to store a cpu_set_t.

// Use match statement for clearer control flow especially when pid == 0 logic is implemented.

// Check if the process exists, in a Rust idiomatic way using match for clarity.

// Assuming all CPUs are available; construct the cpu_set_t accordingly.
```

Reply:
Done.